sections/dashboard/SpotHistoryTable/SpotHistoryTable.tsx

```diff
@@ -251,6 +262,22 @@ const StyledText = styled.div`
 	color: ${(props) => props.theme.colors.common.secondaryGray};
 `;
 
+const TableNoResults = styled(GridDivCenteredRow)`
+	padding: 50px 0;
+	justify-content: center;
+	margin-top: -2px;
+	justify-items: center;
+	grid-gap: 10px;
+	color: ${(props) => props.theme.colors.selectedTheme.button.text};
+	font-size: 20px;
+	font-family: ${(props) => props.theme.fonts.bold};
+	div {
+		text-decoration: underline;
+		font-size: 16px;
+		font-family: ${(props) => props.theme.fonts.regular};
+	}
```

Comment:
Please add `cursor: pointer;` into `div {...}`.
Same css change for the `FutureHistoryTable.tsx` (it's not your problem but it's good that you can fix it in this PR).